doc/user/upgrade/upgrade_guide.md

```diff
@@ -22,6 +22,30 @@ $ kubectl edit deployment/etcd-operator
 ### Incompatible upgrade
 In the case of an incompatible upgrade, the process requires restoring a new cluster from backup. See the [incompatible upgrade guide](incompatible_upgrade.md) for more information.
 
+## v0.6.0 -> v0.6.1
+
+**Note:** If your cluster does not have the following backup policy then you can simply upgrade the operator to the `v0.6.1` image.
+
+Backup policy:
+- A PV storage type: `backup.StorageType=PersistentVolume`
+- No storage class specified: `backup.pv.StorageClass` is unset
+
+In releases prior to `v0.6.1` the etcd-operator would by default create a new storage class for the above backup policy. The `Provisioner` field for that storage class would be set to the vaule of the `--pv-provisioner` flag(`kubernetes.io/gce-pd` or `kubernetes.io/aws-ebs`).
+
+In the `v0.6.1` release the etcd-operator will no longer create a storage class by default. This behavior is set by the new flag `--create-storage-class` whose default value is `false`.
+
+So if your cluster has the above backup policy then do the following steps before upgrading the operator image to `v0.6.1`.
+
+- There should be an existing storage class created by the operator named `etcd-backup-gce-pd` or `etcd-backup-aws-ebs` depending on the value of your `--pv-provisioner` flag. Confirm the name of your existing storage class by inspecting the persistent volume claim for your cluster. Look at the vaule of the `STORAGECLASS` column for the following command. Replace `<cluster-name>` with your cluster name:
```

Comment:
~~There should be an existing storage class created by the operator named `etcd-backup-gce-pd` or `etcd-backup-aws-ebs` depending on the value of your `--pv-provisioner` flag.~~

Confirm the name of storage class for a given cluster:

```sh
  kubectl -n e2e-local get pvc -l=etcd_cluster=<cluster-name>
  ... maybe we should use grep or something to get the STORAGECLASS column..
```